src/knx/memory.cpp

```diff
@@ -29,38 +29,68 @@ void Memory::readMemory()
 
     _freeList = new MemoryBlock(flashStart + metadataBlockSize, flashSize - metadataBlockSize);
 
+    uint16_t version = 0;
+    const uint8_t* buffer = popWord(version, flashStart);
```

Comment:
I think this should be apiVersion here and version later, because apiVersion is written first in writeMemory.

Reply:
You are right, fixed.